webpack.config.js

```diff
@@ -26,6 +26,9 @@ module.exports = (env, argv) => {
       chunkFilename: '[name].[contenthash:8].js'
     },
     performance: { hints: false },
+    stats: {
+      loggingDebug: ["sass-loader"],
```

Comment:
Added this so that in the future I can see variable values and expressions.
https://sass-lang.com/documentation/at-rules/debug
We can get rid of this if you'd like, but it might be nice to have in the future.

Reply:
As a general rule, I'm not in favor of adding complexity to our configuration just in case we might need it down the road, as left unchecked that could easily lead to bloated kitchen-sink configurations. That said, I haven't looked into the specifics of this particular issue so I will defer to @scytacki on the question of whether it makes sense to leave this in and enabled, leave it in but commented out, or to remove it entirely.

Reply:
Added you as a reviewer @scytacki . We can just take this out and I will have it copied in my engineering journal for reference. Hows that sound?